application/testing/CMakeLists.txt

```
@@ -758,11 +760,13 @@ if(F3D_PLUGIN_BUILD_EXODUS)
   f3d_test(NAME TestTimeRangeLessThanZeroNegativeAnimationTime DATA negative_range_animated.e ARGS -s --load-plugins=exodus --animation-time=-3)
```

```suggestion
  f3d_test(NAME TestTimeRangeLessThanZeroNegativeAnimationTime DATA negative_range_animated.e ARGS -s --load-plugins=hdf --animation-time=-3)
```

Many other like that in this file